docker/images/pinot/Dockerfile

```diff
@@ -32,9 +32,19 @@ ENV PINOT_BUILD_DIR=/opt/pinot-build
 
 # extra dependency for running launcher
 RUN apt-get update && \
-    apt-get install -y --no-install-recommends vim wget curl git automake bison flex g++ libboost-all-dev libevent-dev libssl-dev libtool make pkg-config && \
+    apt-get install -y --no-install-recommends vim wget curl git automake bison flex g++ libboost-all-dev libevent-dev
+    libssl-dev libtool make pkg-config sysstat linux-perf openjdk-11-dbg && \
```

Comment:
these should be in same line or a '\' at the line end of line 35

Reply:
good catch